### ISSUE TITLE
2htdp/universe: "rshift" key event breaks key=?

### ISSUE THREAD

**Issue by dvanhorn** [samth/test-bugs#171]
###### Originally submitted on: Thu Jan 19 17:48:01 -0500 2012

``` racket
Big-bang generates "rshift" key events when the right shift key is pressed,
but "rshift" is not a key event according to key-event?, so any code that
uses key=? to compare key events will break when right shift is pressed,
giving:

key=?: expects a KEY-EVTS as first argument, given "rshift"
```
##### Steps to Reproduce:

``` racket
Run this program and press the right-side shift key (tested on Mac OS X Lion):

#lang racket
(require 2htdp/universe)
(require 2htdp/image)

(big-bang 0
          (on-key (λ (_ ke) (key=? ke ke)))
          (to-draw (λ (_) (empty-scene 10 10))))
```
##### Release:

```
5.2.1.3--2012-01-12(01ab29e/g)
```
##### Environment:

```
macosx "Darwin doom.local 11.1.0 Darwin Kernel Version 11.1.0: Tue Jul 26 16:07:11 PDT
 2011; root:xnu-1699.22.81~1/RELEASE_X86_64 x86_64" (x86_64-macosx/3m) (get-display-depth)
 = 32
Human Language: english
(current-memory-use) 774995920
Links: (links) = ("assignments" "book" "class"); (links #:user? #f) = (); (links #:root?
 #t) = (); (links #:user? #f #:root? #t) = ()


Collections:
("/Users/dvanhorn/Library/Racket/5.2.1.3/collects"
 (non-existent-path))
("/Users/dvanhorn/Documents/git/racket/collects"
 (".gitignore" "2htdp" "algol60" "at-exp" "browser" "combinator-parser" "compiler"
 "config" "data" "datalog" "db" "defaults" "deinprogramm" "drracket" "drscheme" "dynext"
 "embedded-gui" "eopl" "errortrace" "ffi" "file" "framework" "frtime" "games" "graphics"
 "gui-debugger" "guibuilder" "handin-client" "handin-server" "help" "hierlist" "honu"
 "htdp" "html" "icons" "images" "info-domain" "lang" "launcher" "lazy" "macro-debugger"
 "make" "meta" "mred" "mrlib" "mysterx" "mzcom" "mzlib" "mzscheme" "net" "openssl"
 "parser-tools" "picturing-programs" "plai" "planet" "plot" "preprocessor" "profile"
 "r5rs" "r6rs" "racket" "racklog" "rackunit" "raco" "reader" "readline" "redex"
 "repo-time-stamp" "rnrs" "s-exp" "scheme" "schemeunit" "scribble" "scribblings"
 "scriblib" "setup" "sgl" "sirmail" "slatex" "slideshow" "srfi" "stepper"
 "string-constants" "swindle" "syntax" "syntax-color" "teachpack" "test-box-recovery"
 "test-engine" "tests" "tex2page" "texpict" "trace" "typed" "typed-racket" "!
 typed-scheme" "unstable" "version" "web-server" "wxme" "xml" "xrepl"))

Computer Language: (("Determine language from source") (#(#t print mixed-fraction-e #f #t
 test-coverage) (default) #() "#lang racket\n" #f #f))
```

**Comment by racket-bug-submit** [comment]
###### On Thu, 19 Jan 2012 17:59:52 -0500, matthias at racket-lang dot org assigned this bug to matthias

A commit by matthias@racket-lang.org has resolved this report
  http://git.racket-lang.org/plt/commit/2a34cbfb85

**Comment by racket-bug-submit** [comment]
###### On Thu, 19 Jan 2012 17:59:52 -0500, matthias at racket-lang dot org wrote:

A commit by matthias@racket-lang.org has resolved this report: 2a34cbfb85
